pkg/dataMover/data_mover.go

```diff
@@ -101,6 +112,13 @@ func (this *DataMover) CopyToRepo(peID astrolabe.ProtectedEntityID) (astrolabe.P
 	this.RegisterOngoingUpload(peID, cancelFunc)
 
 	log.Debugf("Ready to call s3 PETM copy API for local PE")
+	if backupRepository.Name != "Not use" {
+		s3PETM, err := utils.GetRepositoryFromBackupRepository(backupRepository, log)
+		if err != nil {
+			log.Errorf("Failed to get S3 repository from backup repository %s", backupRepository.Name)
```

Comment:
Why is this continuing on?  This should be returning the err here, the s3PETM is not valid